Maestro/subscriptions.json

```diff
@@ -30,11 +30,11 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 3159
     },
-    // A build definition that will trigger the dotnet-nightly Docker image build in Docker Hub
-    "build-dotnet-docker-nightly": {
+    // A build definition capable of running any .ps1 script in the dotnet-docker-nightly repo
+    "dotnet-docker-nightly-general": {
       "vsoInstance": "mseng.visualstudio.com",
       "vsoProject": "dotnetcore",
-      "buildDefinitionId": 3590
```

Comment:
Are you deleting this old build def?  Why not use the same build def and just update it?